IAM: Remove the LDAP reloading endpoint from Admin API (#117473)
grafana/grafana: docs/sources/developer-resources/api-reference/http-api/admin.md

## Reload LDAP configuration

{{% admonition type="warning" %}}

> This endpoint has been made obsolete in Grafana 12.4.0.

Reviewer: ```suggestion
Starting with Grafana 12.4.0., this endpoint is removed. 
```

{{% /admonition %}}

> This endpoint has been removed from the Admin API. Starting with Grafana 12.4.0, SSO settings automatically reload the LDAP settings from the database or from the .toml file.

Reviewer 2: Ah thanks for the explanation!
